exonum/src/node/consensus.rs

```diff
@@ -309,13 +337,19 @@ impl NodeHandler {
             if our_block_hash != block_hash {
                 panic!(
                     "Full propose: wrong state hash. Either a node's implementation is \
-                     incorrect or validators majority works incorrectly"
+                     incorrect or validators majority works incorrectly. Calculated hash \
+                     is {:?}, but hash in the propose is {:?}",
```

Comment:
AFAIK, `block_hash` is not included in proposals (and is not computed at the time a proposal is created). Instead, `block_hash` is mentioned in precommits.

Reply:
Yeah, I forgot to fix the note.